modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/frontend/data/set/filter/VocabularyAssetTypesSelectFDSFilter.java

```diff
@@ -21,10 +21,10 @@
  * @author Noor Najjar
  */
 @Component(service = FDSFilter.class)
-public class VocabularyAssetTypesSelectionFDSFilter
+public class VocabularyAssetTypesSelectFDSFilter
```

Comment:
I think this one should end with `SelectionFDSFilter` because it extends the `BaseSelectionFDSFilter`